src/test/java/guitests/MenuControlTest.java

```diff
@@ -21,6 +22,8 @@ public void menuControlTest() {
         modelUpdatedEventTriggered = false;
         UI.events.registerEvent((ModelUpdatedEventHandler) e -> modelUpdatedEventTriggered = true);
         PanelControl panelControl = (PanelControl) find("#dummy/dummy_col0").getParent();
+        Preferences testPref = panelControl.getPreferences();
```

Comment:
That will result in non-static reference error. Can I move the ```getPrefs()``` from ```PanelControl``` to ```UI``` for this purpose?